CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 * Add List.move extension method that moves an element from one index to another. Delegates to ManagedRealmList.move for managed lists. This allows notifications to correctly report moves, as opposed to reporting moves as deletes + inserts. ([#1037](https://github.com/realm/realm-dart/issues/1037))
 * Support setting `shouldDeleteIfMigrationNeeded` when creating a `Configuration.local`. ([#1049](https://github.com/realm/realm-dart/issues/1049))
 * Add `unknown` error code to all SyncErrors: `SyncSessionErrorCode.unknown`, `SyncConnectionErrorCode.unknown`, `SyncClientErrorCode.unknown`, `GeneralSyncErrorCode.unknown`. Use `unknown` error code instead of throwing a RealmError. ([#1052](https://github.com/realm/realm-dart/pull/1052))
+* Support `RealmValue` that allows users to store different types in the same field. ([#1051](https://github.com/realm/realm-dart/pull/1051))
```

Comment:
It would be better if we add more details here like what .NET did https://github.com/realm/realm-dotnet/blob/main/CHANGELOG.md?plain=1#L825

Reply:
Fixed